myokit/formats/sbml/_importer.py

```diff
@@ -371,39 +1056,43 @@ class SBMLError(myokit.ImportError):
     """
 
 
-unit_map = {
-    'dimensionless': myokit.units.dimensionless,
+# SBML base units according to libSBML (order matters for unit identification!)
```

Comment:
How does order matter for identification?

Reply:
I forgot to remove that comment. It mattered before, because with libSBML the units were numbered and I used an DictKey array to access the keys. 
But now it doesn't matter anymore, because we are using keys directly.